pkg/apis/core/types_common.go

```diff
@@ -47,6 +47,8 @@ const (
 	ErrorConfigurationProblem ErrorCode = "ERR_CONFIGURATION_PROBLEM"
 	// ErrorRetryableConfigurationProblem indicates that the last error occurred due to a retryable configuration problem.
 	ErrorRetryableConfigurationProblem ErrorCode = "ERR_RETRYABLE_CONFIGURATION_PROBLEM"
+	// ErorUserWebhook indicates that the last error occurred due to misconfigurred webhook.
```

Comment:
```suggestion
	// ErorUserWebhook indicates that the last error occurred due to misconfigured webhook.
```
Please change at other places too.